desktop/src/main.rs

```diff
@@ -109,6 +110,10 @@ struct Opt {
     #[clap(long, short, default_value = "show-all")]
     scale: StageScaleMode,
 
+    /// Audio volume
```

Comment:
Let's add a little more information here - something like this:
`/// The audio volume as a number between 0 (muted) and 1 (full volume)`